Template/azuredeploy.json

```diff
@@ -177,11 +187,27 @@
                         },
                         {
                             "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
-                            "value": "[reference(concat('microsoft.insights/components/', variables('appInsightName'))).InstrumentationKey]"
+                            "value": "[if(parameters('useAzureMonitor'), reference(concat('microsoft.insights/components/', variables('appInsightName'))).InstrumentationKey, '')]"
```

Comment:
I am wondering here, someone doesn't want app insight monitoring the edge, but he does want that for the function. For example the use case we are currently working on. Let's discuss about the scenarios

Reply:
Documenting what we discussed offline: I will revert back to the original state: monitoring in the cloud can never be turned off (AppInsights will always be deployed). The switch will only apply to monitoring on the edge.

Reply:
Solved in 4a7615f240aa978ff70b375465e021678bfdde6f